docs/README.md

```diff
@@ -1,56 +1,102 @@
 # What is cmdalias
 
-`cmdalias` is a tool that I created for myself because I was tired of using too many keystrokes for commands that i'm using everyday in my shell. For example I often want to list images from my docker.
+`cmdalias` is an alias tool inspired from git-config that I created for myself. Moreover, it lets you create
```

Comment:
I would say that it's inspired by the git-config alias mechanisme more than by git-config. Git-config is not just handling aliases.